### DISCUSSION TITLE
In React Native It Force close Application

### DISCUSSION THREAD

**Discussion by pranavithape** [nodejs-mobile/nodejs-mobile#115] [General]
Hey I did everything as tutorial said

- Installing plain react native
- then i setup emulator 
- then i installed everything needed like jdk etc to check everything okay and working right and aslo app is launching well without or before installing package
- but when i install package and open android folder in andorid studio to sync gradle etc
- then i setup files as you said main and package json copy them add then in start function
- The problem is when i do this while emulator open and do changes in code and added your start and use effect code and add your button element i recive this error - cannot read property 'sendmessage' null idont know
- and even i try to restart this time app dosnt open it closing instantly i tried many this would anyone help me if they have similar issues

**Comment by gev2002** [comment]
Hi, you can use nodejs-mobile for apps with react-native version below 0.74. The react-native 0.74 working  with new architecture, but the module cannot support this until it is updated.

**Reply by pranavithape** [reply]
Hey Budy Thanks For Your Replay I Glad About This Entire System And People Like You , I Fixed My Error Somehow Its Development Envirment Setup Wrong LIke Stuff Android Studio Envirment Variable Etc Etc , One Thing I If you can help how should modules support are if i am correct we are talking about external liabraries found on npm for node -- Thank You
